packages/cli/README.md

```diff
@@ -114,7 +126,7 @@ ARGUMENTS
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Private Key
```

Comment:
I assume that no read-only commands ever need --key

Reply:
I think this should stay as it allows a user to load their DID based on a seed.